app/services/prophet_ratings/game_finalizer.rb

```
@@ -0,0 +1,85 @@
+        away_team_snapshot: away_snapshot
+      )
+      return unless prediction
+
+      prediction.update!(
+        home_offensive_efficiency_error: prediction.home_offensive_efficiency - game.home_team_game.offensive_efficiency,
+        away_offensive_efficiency_error: prediction.away_offensive_efficiency - game.away_team_game.offensive_efficiency,
+        home_defensive_efficiency_error: prediction.away_offensive_efficiency - game.away_team_game.offensive_efficiency,
+        away_defensive_efficiency_error: prediction.home_offensive_efficiency - game.home_team_game.offensive_efficiency,
+        pace_error: prediction.pace - game.pace
+      )
+    end
```

RuboCop (1.75.5)

[convention] Assignment Branch Condition size for `finalize_prediction!` is too high. [<1, 30, 1> 30.03/22]

(Metrics/AbcSize)